eng/versioning/version_client.txt

```diff
@@ -58,7 +58,7 @@ com.azure:azure-communication-sms;1.0.3;1.1.0-beta.1
 com.azure:azure-communication-identity;1.1.1;1.2.0-beta.1
 com.azure:azure-communication-phonenumbers;1.0.3;1.1.0-beta.1
 com.azure:azure-containers-containerregistry;1.0.0-beta.3;1.0.0-beta.4
-com.azure:azure-core;1.18.0;1.19.0-beta.2
+com.azure:azure-core;1.19.0-beta.1;1.19.0-beta.2
```

Comment:
Can also use a `beta_com.azure:azure-core;1.19.0-beta.1` tag

Reply:
Added - I suppose we could do this in the `main` branch too?

Reply:
Saw it https://github.com/Azure/azure-sdk-for-java/blob/main/eng/versioning/version_client.txt#L328-L335

Seems added recently.
If we can use that, we can go for main directly for purview account preview release.

Reply:
Yes - closing this PR then.